### DISCUSSION TITLE
Import songs sorted with stable's "Last Played" sort

### DISCUSSION THREAD

**Discussion by EXtremeExploit** [ppy/osu#18012] [Ideas]
Derived from https://github.com/ppy/osu/discussions/13713, also of doing date added, would be nice to also import last played which is stored on legacy osu!.db. Sadly lazer doesn't store last played by each beatmap nor beatmap set, so some changes would have to be made afaik
So this discussion could be branched into 2 separate issues: Add option to sort by Last played and make import from stable also import last played time.
No last played option as seen in the screenshot
![image](https://user-images.githubusercontent.com/30161277/165814636-4aaafd87-a0f1-458e-a0b6-8d218540f248.png)

**Comment by peppy** [comment]
https://github.com/ppy/osu/issues/13717

**Reply by frenzibyte** [reply]
This thread tracks sorting by the last time the beatmap was played, though? unlike #13717/#13713.

**Reply by EXtremeExploit** [reply]
correct, this is for last played time, not date added

**Reply by peppy** [reply]
Ah, I think I mistook the thread sorry.
